api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
@pytlesk4, at the beginning this lib was started as a transparent shim between C and Go. Due to that fact that in Go some things could be done in a more elegant way, I would say that we can accept some trade-offs for more convenience. 

In this certain case, I would return the value instead of passing a pointer b/c it's more safety. Also, do we need to return a length of the buffer if we have the buffer? It seems no.

Also, I do not agree with you about test package name. It's very much ok if we will have some examples right at the godoc page. If we split test packages(which is non-idiomatic at all) we will lose this ability.

@mosolovsa ask you to revert test package to `duktape`(at it was suggested previously) and fix buffers/strings lengths. I think we shouldn't return it b/c we have the value which could be measured. 

Correct my guys if you think I am wrong. Open to discussion.